Package.swift

```diff
@@ -14,17 +14,17 @@ let package = Package(
     dependencies: [
       .package(name: "mParticle-Apple-SDK",
                url: "https://github.com/mParticle/mparticle-apple-sdk",
-               .upToNextMajor(from: "8.22.0")),
-      .package(name: "Adjust",
+               .upToNextMajor(from: "8.26.0")),
```

Comment:
```suggestion
               .upToNextMajor(from: "8.22.0")),
```
While I do like to promote upgrading the core SDK, if there's no dependency on the newer version we shouldn't bump this. Also the podspec still references 8.22.0 and up so this should match